### DISCUSSION TITLE
How can I contribute to add support for my model ?

### DISCUSSION THREAD

**Discussion by mooms06** [BartoszCichecki/LenovoLegionToolkit#1139] [Q&A] [answered]
My laptop, an IdeaPad Pro 5 16APH8, is partially compatible with LLT, the only critical feature that is missing is switching the performance plans like Vantage. I would like to get rid of Vantage to completely replace it with LLT but I can't right now.

![image](https://github.com/BartoszCichecki/LenovoLegionToolkit/assets/25405517/ca388d03-06cf-437e-86f3-cb9a20e48e51)

I'm not a dev, but I'm tech in IT so I can perform any needed  tasks if guided.

Thank you.

**Comment by BartoszCichecki** [comment, marked as answer]
The readme is very clear on that matter. I don't have plans on adding support for other devices. If some one wants to implement it, go ahead, but I don't have time or means to do it.
